myRental/test/test.js

```diff
@@ -44,27 +49,91 @@ describe('Database Module Test', () => {
     });
 
     describe('post -> /login', () => {
-        it('A user should be created and also be able to login', () => {
+        it('A user should be created and also be able to login', (done) => {
             database.registerUser(testUserRequest, (callBack) => {
                 database.login(testUserRequest, (callBack) => {
                     assert.equal(callBack.success, true);
+                    done();
                 });
             });
         });
     });
   });
 });
 
+describe('requestController', () => {
+
```

Comment:
Rogue newline

Reply:
That was intentional, that describe statement could be more multiple tests so I like a line between it and the specific describes and it() tests.